Add missing link (#7553)
DataDog/documentation: content/en/account_management/rbac/permissions.md

Grant the following permissions to manage read access on subsets of log data:

* `logs_read_data`(Recommended) offers finer grained access control by restricting a role's access to logs matching a log restriction queries. 

> * `logs_read_index_data` is the alternative approach to restrict data access to indexed log data on a per-index basis (it is still required to have this permission in order to access indexed data)

Reviewer: ```suggestion
* `logs_read_index_data` is the alternative approach to restrict data access to indexed log data on a per-index basis (it is still required to have this permission to access indexed data).
```

#### logs_read_index_data

Grants a role read access on some number of log indexes. Can be set either globally or limited to a subset of log indexes.

> When using `logs_read_data` and restriction queries, `logs_read_index_data` permission **must** be set globally in order to access indexed logs.

Reviewer: ```suggestion
When using `logs_read_data` and restriction queries, the `logs_read_index_data` permission **must** be set globally to access indexed logs.
```

[1]: /account_management/users/#edit-a-user-s-roles
[2]: /api/v2/roles/#list-permissions

> [4]: /api/v2/logs-restriction-queries/

Reviewer: Change this to `[3]` and the corresponding reference.

Author: Done :)
